model/Core/Properties/definingDocument.md

```diff
@@ -4,15 +4,15 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-URI for an SPDX document which defines an SPDX element.
+Artifact containing SPDX data which defines one or more SPDX elements.
```

Comment:
Suggest small tweak:

"Artifact representing a serialization instance of SPDX data containing the definition of a particular Element."

Reply:
Updated per suggestion